orchestra/cmds/graph.py

```diff
@@ -52,7 +54,7 @@ def handle_graph(args):
             else:
                 actions.add(component.default_build.install)
 
-    executor = Executor(args, actions)
+    executor = Executor(args, actions, no_force=args.no_force)
```

Comment:
Is this good also for `orc clone`? This would simplify `ci.sh`.

Reply:
Yep, I added `--no-force` to `clone` as well, now `ci.sh` no longer needs to check if the sources were already cloned, it can just use `orc clone --no-force`